Source/WebCore/platform/graphics/SourceBufferPrivate.cpp

```diff
@@ -400,8 +405,7 @@ void SourceBufferPrivate::removeCodedFramesInternal(const MediaTime& start, cons
     // 3. For each track buffer in this source buffer, run the following steps:
 
     iterateTrackBuffers([&](auto& trackBuffer) {
-        trackBuffer.removeCodedFrames(start, end, currentTime);
-
+        m_evictionData.contentSize -= trackBuffer.removeCodedFrames(start, end, currentTime);
```

Comment:
Is there a chance we somehow mess up the computation and have contentSize go below 0 and turn huge?
Should we instead just set contentSize to totalTrackBufferSizeInBytes  after this iteration?

Reply:
this method is called in a loop from the eviction algorithm. The data can't roll-over and I have placed assert in the TrackBuffer indicating as such.

As soon as the loop is complete, things will be recalculated. The partial recalculation is only used while in the loop of `SourceBuffer:evictFrames`